src/legacy/core_plugins/kibana/public/discover/angular/discover.js

```diff
@@ -419,6 +423,7 @@ function discoverController(
     queryFilter.setFilters(filters);
   };
 
+  // TODO this isnt used anymore here, just in visualize and dashboards
   $scope.applyFilters = filters => {
```

Comment:
This can be definitely deleted - @ppisljar deprecated this in a recent PR.
No reason to keep it around.